src/dotnet/APIView/APIViewUnitTests/CodeFileHelpersTest.cs

```diff
@@ -322,5 +322,25 @@ public async void VerifyNavigationNodeCount()
             var navTreeNodeCount = result.NodeMetaData.Values.Count(n => n.NavigationTreeNode != null);
             Assert.Equal(42, navTreeNodeCount);
         }
+
+
+        [Fact]
+        public async void VerifyJavaPackageReviewLineCount()
```

Comment:
in theory this should apply to every language?
if so, consider changing the name to something more generic